### PR TITLE
added progress bar stories

### PR DESCRIPTION
**Pull request by TITHI1244**
# Description

Added a progress bar storybook component.

Fixes # (issue)
#9 

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by bytewiz — APPROVED** [review]
Looks good but be aware of global styling

File: src/client/components/ProgressBar.components/ProgressBar.stories.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    />
+  );
+};
+
+export const Progress = Template.bind({});
+Progress.args = {
+  level: '20',
+  backgroundColor: 'red',
+  alphaLevel: '0.5',
+};
```
Comment (bytewiz) [inline src/client/components/ProgressBar.components/ProgressBar.stories.js:35]: You have to add this for the `propTypes` warnings :) 
image

File: src/client/components/ProgressBar.components/ProgressBar.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
[the remark this answers is not in the document]
Comment (TITHI1244) [reply src/client/components/ProgressBar.components/ProgressBar.css]: Yes, fixed it.

File: src/client/components/ProgressBar.components/ProgressBar.stories.js (hunk 1 above)
Comment (TITHI1244) [reply src/client/components/ProgressBar.components/ProgressBar.stories.js:35]: Got that, thanks

**Review by Enqira — APPROVED** [review]
reviewed and changes were well implemented, ready to merge

**Pull request merged**